File: src/main/java/org/trackedout/citadel/classes/Party.java
package org.trackedout.citadel.classes;

import java.util.ArrayList;

import org.bukkit.entity.Player;

public class Party {
	public Player leader = null;
	public ArrayList<Player> members = new ArrayList<Player>();

	public Party(Player leader) {
		this.leader = leader;
		this.members.add(leader);
	}

	public void addMember(Player member) {
		this.members.add(member);
	}

	public void removeMember(Player member) {
		this.members.remove(member);
	}

}

File: src/main/kotlin/org/trackedout/citadel/data/Party.kt
package org.trackedout.citadel.data

import org.bukkit.entity.Player
import java.util.*

class Party(var leader: Player, var members: ArrayList<Player> = ArrayList<Player>()) {
    init {
        members.add(leader)
    }

    fun addMember(member: Player) {
        members.add(member)
    }

    fun removeMember(member: Player) {
        members.remove(member)
    }
}
